src/components/input/OrganizationSelectInput/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useMemo } from 'react';
+import { SearchSelectInput, SearchSelectInputProps } from '@the-deep/deep-ui';
+import { Organization, MultiResponse } from '#typings';
+
+import { useRequest } from '#utils/request';
+import useDebouncedValue from '#hooks/useDebouncedValue';
+import _ts from '#ts';
+import { notifyOnFailure } from '#utils/requestNotify';
+
+type Def = { containerClassName?: string };
+type OrganizationSelectInputProps<K extends string> = SearchSelectInputProps<
+    number,
+    K,
+    Organization,
+    Def,
+    'onSearchValueChange' | 'searchOptions' | 'optionsPending' | 'keySelector' | 'labelSelector' | 'totalOptionsCount'
+>;
+const keySelector = (d: Organization) => d.id;
+const labelSelector = (d: Organization) => d.title;
+function OrganizationSelectInput<K extends string>(props: OrganizationSelectInputProps<K>) {
+    const {
+        className,
+        ...otherProps
+    } = props;
+
+    const [searchText, setSearchText] = useState<string>('');
+    const debouncedSearchText = useDebouncedValue(searchText);
+
+    const searchQueryParams = useMemo(() => ({
+        search: debouncedSearchText,
+    }), [debouncedSearchText]);
+
+    const {
+        pending: organizationSearchPending,
+        response: organizations,
+    } = useRequest<MultiResponse<Organization>>(
+        {
+            url: 'server://organizations/',
+            method: 'GET',
+            query: searchQueryParams,
+            onFailure: (_, errorBody) =>
+                notifyOnFailure(_ts('project.users', 'organizationTitle'))({ error: errorBody }),
```

Comment:
```suggestion
                notifyOnFailure(_ts('components', 'organizationTitle'))({ error: errorBody }),
```
Lets scope it under something other than project.users